app/serializers/v1/GroupSerializer.js

```diff
@@ -2,6 +2,6 @@ var Serializer = require("../../models").Serializer
 
 exports.addSerializer = function() {
   return new Serializer('groups', {
-    select: ['id', 'username', 'type', 'screenName']
+    select: ['id', 'username', 'type', 'screenName', 'visibility', 'postsTimelineId']
```

Comment:
I can't stop thinking that postsTimelineId is redundant here (i.e. user never submits a post to a timeline and never works with timelines directly, but to a group name). Like we can upgrade PostsController#create method to accept feed names (that are uniq, btw). 

On the one hand we'll have to select postsTimelineId manually with an extra request to a database:
```
var feeds = req.body.feeds
Promise.map(feeds, function(username) {
  FeedFactory.findByUsername(username)
    .then(function(feed) { return feed.getPostsTimelineId })
})
.then(function(timelineIds) {
  <do the rest>
})

However on the other this will make app more robust and secure (timeline hijack).